src/test/java/io/javalin/examples/HelloWorldCompression.java

```diff
@@ -0,0 +1,46 @@
+package io.javalin.examples;
+
+import io.javalin.Javalin;
+import io.javalin.core.compression.Brotli;
+import io.javalin.core.compression.DynamicCompressionStrategy;
+import io.javalin.core.compression.Gzip;
+
+import java.util.*;
+
+public class HelloWorldCompression {
```

Comment:
I don't think this should be part of the final patchset.

Reply:
No problem, will get rid of it!